showkase-processor/src/main/java/com/airbnb/android/showkase/processor/ShowkaseProcessor.kt

```diff
@@ -44,14 +46,43 @@ class ShowkaseProcessor @JvmOverloads constructor(
     private val logger = ShowkaseExceptionLogger()
     private val showkaseValidator = ShowkaseValidator()
 
-    override fun getSupportedAnnotationTypes(): MutableSet<String> = mutableSetOf(
-        ShowkaseComposable::class.java.name,
-        PREVIEW_CLASS_NAME,
-        ShowkaseColor::class.java.name,
-        ShowkaseTypography::class.java.name,
-        ShowkaseRoot::class.java.name,
-        ShowkaseScreenshot::class.java.name,
-    )
+    override fun getSupportedAnnotationTypes(): MutableSet<String> = supportedAnnotationTypes()
+
+    private fun supportedAnnotationTypes(): MutableSet<String> {
+        val set = mutableSetOf<String>(
+            ShowkaseComposable::class.java.name,
+            PREVIEW_CLASS_NAME,
+            ShowkaseColor::class.java.name,
+            ShowkaseTypography::class.java.name,
+            ShowkaseRoot::class.java.name,
+            ShowkaseScreenshot::class.java.name,
+        )
+
+        environment
+            .options["MultiPreviewTypes"]
+            ?.split(",")?.map { it.replace(" ", "") }
+            ?.toSet()?.let { set.addAll(it) }
+        return set
+    }
+
+    private fun getSupportedMultipreviewTypes(): Set<String> {
```

Comment:
move these two functions below where it's actually being used, that way it's easier to follow in code for future code reviews

Reply:
Good idea :) Moved them in [50bf2f5](https://github.com/airbnb/Showkase/pull/263/commits/50bf2f500ce613997694172deeea1e0459d7e14b)